linera-execution/src/execution.rs

```diff
@@ -153,30 +153,27 @@ where
         oracle_record: Option<OracleRecord>,
         resource_controller: &mut ResourceController<Option<Owner>>,
     ) -> Result<(Vec<ExecutionOutcome>, OracleRecord), ExecutionError> {
-        let (execution_outcomes, oracle_record) =
-            match self.context().extra().execution_runtime_config() {
-                ExecutionRuntimeConfig::Synchronous => {
-                    self.run_user_action_with_synchronous_runtime(
-                        application_id,
-                        chain_id,
-                        local_time,
-                        action,
-                        refund_grant_to,
-                        grant,
-                        oracle_record,
-                        resource_controller,
-                    )
-                    .await?
-                }
-            };
+        let ExecutionRuntimeConfig {} = self.context().extra().execution_runtime_config();
```

Comment:
Do we still need this line? 🤔 same for below

Reply:
It's nice that the typechecker will break here when we add a parameter.

Reply:
This being said, perhaps, we should ask ourself if we intend to add such parameter.

Reply:
Ah right, good point

Reply:
It's a placeholder for when we add configuration parameters (as fields) to the type. We could also remove everything, but then we'd need to revert it later, so it seems like less work to leave it like this temporarily :sweat_smile: